databricks/sdk/core.py

```diff
@@ -91,12 +91,24 @@ def inner() -> Dict[str, str]:
 @credentials_provider('runtime', [])
 def runtime_native_auth(cfg: 'Config') -> Optional[HeaderFactory]:
     from databricks.sdk.runtime import init_runtime_native_auth
-    if init_runtime_native_auth is None:
-        return None
-    else:
+    if init_runtime_native_auth is not None:
         host, inner = init_runtime_native_auth()
         cfg.host = host
         return inner
+    try:
+        from dbruntime.databricks_repl_context import get_context
```

Comment:
Hi @nfx may I know whether you verify in notebook environment (e.g. with DBR 12.2) that it works? I vaguely remember last time I want to import from `dbruntime` in `core.py` it does not work. Ignore me if I'm wrong.

Reply:
I've tested it on couple of DBRs